src/test/skript/tests/regressions/6881-sorted indices with children.sk

```diff
@@ -0,0 +1,9 @@
+test "sorted indices on map":
+	set {_test::1} to 111
+	set {_test::2} to 555
+	set {_test::3} to 444
+	set {_test::1::a} to 2
+	set {_test::2::b} to 3
+	set {_test::3::c} to 6
```

Comment:
I'd like to see a test with 3 levels, just as a safety measure, otherwise looks good.